apps/molgenis-components/src/components/tables/TableMolgenis.vue

```diff
@@ -184,7 +184,10 @@ export default {
       const key = this.getPrimaryKey(row);
       if (this.showSelect) {
         //deep copy
-        let update = deepClone(this.selection);
+        let update = []; //prevent null list
+        if (Array.isArray(this.selection)) {
```

Comment:
Could just do `if (this.selection) { ....` this feels a bit like overengineering ;)